handlers/main.yml

```diff
@@ -6,7 +6,6 @@
   service:
     name: "{{ bambooagent_service_name }}"
     enabled: yes
-    state: reloaded
```

Comment:
is this required?

Reply:
That is what I found, yes. 
With the state specified, the execution of the role will fail when it is run for the first time on the agent. After that, no problem though.
Without it, the execution goes smoothly in all cases.
Tested on a 18.04 VM with ansible 2.8.5